birdhouse/deployment/certbotwrapper

```diff
@@ -50,21 +50,10 @@ THIS_FILE="`realpath "$0"`"
 THIS_DIR="`dirname "$THIS_FILE"`"
 SAVED_PWD="`pwd`"
 
-# Default values
-. $THIS_DIR/../default.env
+. "$THIS_DIR/../read-configs.include.sh"
 
-ENV_LOCAL_FILE="$THIS_DIR/../env.local"
-
-set +x  # hide password in env.local
-
-if [ -e "$ENV_LOCAL_FILE" ]; then
-    # Override default values
-    . "$ENV_LOCAL_FILE"
-else
-    echo "WARNING: '$ENV_LOCAL_FILE' not found."
-fi
-
-set -x  # resume logging/tracing
+# Get PAVICS_FQDN_PUBLIC, PAVICS_FQDN, SUPPORT_EMAIL, SSL_CERTIFICATE.
+read_configs
```

Comment:
In all files where this is done, consider removing the blank lines between the statements, as they are strongly connected. Having the call right after the include line highlights that it is defined in that file.

Reply:
The empty line between the include and the call is because I always comment which vars I try to get with the call so it would look weird to not have an empty line.  Ex:

```
. read-configs.include.sh
# get VAR1, VAR2, VAR3
read_configs
```

I usually put an empty line before a comment.

Also the file is `read-config.include.sh` and the function is `read_configs` so I think the connection is fairly obvious even with a empty line in between.

Still I did remove the empty line when no comment in between, see 2e16242503e2d3471525016cd95a97d011706691